.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,169 @@
+# Contribute to Trinity
```

Comment:
Should we just move `CONTRIBUTING.md` to `.github/CONTRIBUTING.md`?

Reply:
How do you mean?

Reply:
There's already a similar doc called `CONTRIBUTING.md`:  https://github.com/iotaledger/trinity-wallet/blob/develop/CONTRIBUTING.md